src/ros_license_toolkit/common.py

```diff
@@ -18,11 +18,11 @@
 
 from typing import Any, Dict
 
+REQUIRED_PERCENTAGE_OF_LICENSE_TEXT = 95.0
+
 
 def is_license_text_file(scan_results: Dict[str, Any]) -> bool:
     """Check if a file is a license text file."""
-    for _license in scan_results["licenses"]:
-        if _license["matched_rule"]["is_license_text"] and \
-                _license["score"] >= 99:
-            return True
-    return False
+    return (
+        scan_results["percentage_of_license_text"] >=
+        REQUIRED_PERCENTAGE_OF_LICENSE_TEXT)
```

Comment:
Does this even make sense anymore? I think I split it out because it was used in multiple classes. Feel free to remove this file if this does not make sense any more

Reply:
Maybe not for this pull request, but on another branch I added one more function here, thats why I'd keep it :).